app/src/componentLibrary/sidebar/sidebar.jsx

```diff
@@ -32,20 +33,30 @@ export const Sidebar = ({
   topSidebarItems,
   bottomSidebarItems,
   createFooterBlock,
+  isOpenPopover,
```

Comment:
It would be better not to rely on `popover` usage somewhere not in this component. It may be another event or smth that will not allow to close sidebar.
So I suggest renaming it to something more generic, like `shouldBeCollapsedOnLeave`. That way it will be more clear what happened when this prop provided.